spec/json_validation_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'json'
+require 'json_schemer'
+
+describe 'json_validation' do
+  Dir['./**/*.json'].each do |json_file|
+    next if json_file.include? 'schema.json'
+
+    name = File.basename(json_file, '.json')
+    dir = File.dirname(json_file)
+    it json_file.to_s do
+      json_schema_candidates = [
+        "#{dir}/#{name}.schema.json",
+        "#{dir}/schema.json",
```

Comment:
Why will you need these two forms?

Reply:
In the case of platforms Win2022x64, Win2016x64, etc we will have one schema for a lot of files. Same with devices.

Reply:
Makes sense. I added a few more comments for the logic to handle the two forms.